strongbox-commons/src/main/java/org/carlspring/strongbox/booters/PropertiesBooter.java

```diff
@@ -181,4 +187,19 @@ public void setStrongboxNugetDownloadFeed(boolean strongboxNugetDownloadFeed)
         this.strongboxNugetDownloadFeed = strongboxNugetDownloadFeed;
     }
 
+    public String getStrongboxProjectVersion() {
```

Comment:
Cosmetic: Could you please correct the indentation here and across the entire pull? Thanks! :)